File: frontend/src/types/enums/ServiceableVehicle.ts
export enum ServiceableVehicle {
  CAR = "CAR",
  TRUCK = "TRUCK"
}

export function getServiceableVehicleName(serviceableVehicles: ServiceableVehicle): string{
  switch(serviceableVehicles) {
    case ServiceableVehicle.CAR:
      return "Car";
    case ServiceableVehicle.TRUCK:
      return "Truck"
  }
}

File: backend/src/main/java/com/alexisa/tire_reserve/model/domain/enums/ServiceableVehicle.java
package com.alexisa.tire_reserve.model.domain.enums;

import lombok.Getter;

@Getter
public enum ServiceableVehicle {
    CAR("CAR"),
    TRUCK("TRUCK");

    private final String name;

    ServiceableVehicle(String name) {
        this.name = name;
    }
}
